database/sql/V012__create_app_data.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE app_data (
+    app_data_hash bytea PRIMARY KEY,
+    app_code bytea,
+    referrer bytea,
+    file_blob bytea NOT NULL
+);
+-- Get a specific referral.
+CREATE INDEX referrer ON app_data USING BTREE (referrer);
+
+-- Get a specific app_code.
+CREATE INDEX app_code ON app_data USING BTREE (app_code);
```

Comment:
nit: newline

Reply:
Might also want to give the indexes different name than the fields themselves. At least from the docs this appears to be common practice.